Author: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: 2019-08-24
Description: [PATCH] Shutdown of desktop does not need wnck shutdown
 Ubuntu apport catches these wnck crashes as reports these. For a desktop
 shutdown there is no need to catch wnck crashes.
Forwarded: not-needed

---
 src/daemon/main.vala | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/src/daemon/main.vala
+++ b/src/daemon/main.vala
@@ -91,8 +91,9 @@
     manager = null;
     end_dialog = null;
     settings = null;
+    screen = null;
 
-    Wnck.shutdown();
+    //Wnck.shutdown();
 
     return 0;
 }
